ext/openssl/openssl.c

```diff
@@ -7296,7 +7292,7 @@ PHP_FUNCTION(openssl_decrypt)
 	size_t data_len, method_len, password_len, iv_len = 0, tag_len = 0, aad_len = 0;
 	zend_string *ret;
 
-	if (zend_parse_parameters(ZEND_NUM_ARGS(), "sss|lsss", &data, &data_len, &method, &method_len,
+	if (zend_parse_parameters(ZEND_NUM_ARGS(), "sss|lss!s", &data, &data_len, &method, &method_len,
```

Comment:
Is it possible to use `$tag = ""` instead?

Reply:
Yes, it is based on `} else if (!enc && tag && tag_len > 0) {` in `php_openssl_cipher_init()`.